shared/django_apps/reports/migrations/0022_reducederror_testinstance_reduced_error_flake.py

```diff
@@ -0,0 +1,121 @@
+CREATE INDEX "reports_testinstance_reduced_error_id_f90c8b72" ON "reports_testinstance" ("reduced_error_id");
+ALTER TABLE "reports_flake" ADD CONSTRAINT "reports_flake_reduced_error_id_1d102637_fk_reports_r" FOREIGN KEY ("reduced_error_id") REFERENCES "reports_reducederror" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "reports_flake" ADD CONSTRAINT "reports_flake_repoid_1454c21c_fk_repos_repoid" FOREIGN KEY ("repoid") REFERENCES "repos" ("repoid") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "reports_flake" ADD CONSTRAINT "reports_flake_testid_9873bd1c_fk_reports_test_id" FOREIGN KEY ("testid") REFERENCES "reports_test" ("id") DEFERRABLE INITIALLY DEFERRED;
+CREATE INDEX "reports_flake_reduced_error_id_1d102637" ON "reports_flake" ("reduced_error_id");
+CREATE INDEX "reports_flake_repoid_1454c21c" ON "reports_flake" ("repoid");
+CREATE INDEX "reports_flake_testid_9873bd1c" ON "reports_flake" ("testid");
+CREATE INDEX "reports_flake_testid_9873bd1c_like" ON "reports_flake" ("testid" text_pattern_ops);
+CREATE INDEX "reports_fla_repoid_69b787_idx" ON "reports_flake" ("repoid", "testid", "reduced_error_id", "end_date");
+COMMIT;
+"""
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0052_increment_version"),
+        (
+            "reports",
+            "0021_remove_testinstance_flaky_status_and_more",
+        ),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ReducedError",
+            fields=[
+                ("id", models.BigAutoField(primary_key=True, serialize=False)),
+                ("external_id", models.UUIDField(default=uuid.uuid4, editable=False)),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                ("message", models.TextField()),
+            ],
+            options={
+                "db_table": "reports_reducederror",
+            },
+        ),
+        RiskyAddField(
+            model_name="testinstance",
+            name="reduced_error",
+            field=models.ForeignKey(
+                db_column="reduced_error_id",
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="testinstances",
+                to="reports.reducederror",
+            ),
+        ),
+        migrations.CreateModel(
+            name="Flake",
+            fields=[
+                ("id", models.BigAutoField(primary_key=True, serialize=False)),
+                ("external_id", models.UUIDField(default=uuid.uuid4, editable=False)),
```

Comment:
It might be good to sync up and not use `external_id` if we don't need it. (might require model in https://github.com/codecov/shared/pull/252/files#diff-9408bf4ca99f8b12accac14065013a0a52c56db0ba52e5c95729df34a60b1901R16).